src/Workleap.Authentication.ClientCredentialsGrant.Tests/AuthorizationExtensionsTest.cs

```diff
@@ -10,7 +10,7 @@ namespace Workleap.Authentication.ClientCredentialsGrant.Tests;
 public class AuthorizationExtensionsTest
 {
     private const string DefaultAudience = "serviceA";
-    private const string DefaultAuthority = "Https://authority.io";
+    private const string DefaultAuthority = "https://authority";
```

Comment:
I found it a little bit weird to reference an existing website, now it looks more "fake"